_posts/2014-05-14-hacking-bureaucracy-improving-hiring-and-software.html

```diff
@@ -3,7 +3,7 @@
 date: '2014-05-14T11:02:00-04:00'
 tumblr_url: http://18fblog.tumblr.com/post/85724876053/hacking-bureaucracy-improving-hiring-and-software
 
-title: 'Hacking Bureaucracy: improving hiring and software deployment'
+title: "Hacking bureaucracy: improving hiring and software deployment"
```

Comment:
Only if it's a complete sentence after the colon. So "Hacking bureaucracy: We're improving hiring and software deployment" would need the caps.